src/components/Upload.vue

```diff
@@ -1,66 +1,63 @@
 <template lang="pug">
 .upload-wrapper
   slot(name="toolbar")
-    v-toolbar(dense, dark, color="success")
-      v-toolbar-title Upload
+    v-toolbar(dense, dark, flat, color="primary")
+      v-icon cloud_upload
+      v-toolbar-title Upload to
+        v-avatar
+          v-icon {{ ResourceIcons[dest._modelType.toUpperCase()] }}
+        | {{ dest.name }}
```

Comment:
My original reason for splitting this into two rows was that it's more responsive-friendly. In mobile environments, horizontal space is very limited but vertical space is not (since it's expected that scrolling vertically will happen but scrolling horizontally should not).

Reply:
Good point. I was having some trouble looking for an example of a subheader in vuetify/material but I'll move it back to its own line.

Reply:
@jtomeck your thoughts? I also like a single line better, is there something clever we can do to get it on a single line but also convey what action is happening without using too much horizontal space?